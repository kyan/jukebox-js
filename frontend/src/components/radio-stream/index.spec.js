import React from 'react'
import { shallow } from 'enzyme'
import RadioStream from './index'

describe('RadioStream', () => {
  it('plays when active', () => {
    const wrapper = shallow(<RadioStream active />)
    expect(wrapper).toMatchSnapshot()
  })

  it('hides when inactive', () => {
    const wrapper = shallow(<RadioStream active={false} />)
    expect(wrapper).toMatchSnapshot()
  })
})
