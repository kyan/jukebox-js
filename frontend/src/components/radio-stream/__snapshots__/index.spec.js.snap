// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioStream hides when inactive 1`] = `""`;

exports[`RadioStream plays when active 1`] = `
<t
  autoPlay={true}
  className=""
  controls={true}
  controlsList=""
  crossOrigin={null}
  id=""
  listenInterval={10000}
  loop={false}
  muted={false}
  onAbort={[Function]}
  onCanPlay={[Function]}
  onCanPlayThrough={[Function]}
  onEnded={[Function]}
  onError={[Function]}
  onListen={[Function]}
  onLoadedMetadata={[Function]}
  onPause={[Function]}
  onPlay={[Function]}
  onSeeked={[Function]}
  onVolumeChanged={[Function]}
  preload="metadata"
  src="http://icecast.local:8000/radio-kyan.mp3"
  style={Object {}}
  title=""
  volume={1}
/>
`;
